CONTRIBUTING.md

```diff
@@ -909,6 +910,34 @@ test cases.
 you should name it ``module_name_example`` (ie: `cms_form` and `cms_form_example`).
 In this way coverage analysis will ignore this extra module by default.
 
+### Investigating Travis Test Failures
+
+It can sometimes be difficult to reproduce a Travis test failure locally due to
+subtle environment differences. In these scenarios it can be helpful to connect to
+the Runbot container generated for that branch/PR via SSH, where the
+environment will be very similar to Travis. You can do this by running:
+
+```
+ssh -p [port] odoo@runbot[1 or 2].odoo-community.org
```

Comment:
I'll add this. It wasn't working for me, and I wasn't sure why, so I decided to drop it from the instructions, but now I realize it's just because I didn't have a server listening on this port due to the ``--stop-after-init`` flag.